test_django/pseudo_test/views.py

```diff
@@ -0,0 +1,36 @@
+from django.http import HttpResponse
+from django.shortcuts import render
+from .models import Task
+import boto3   
+from test_django.settings import S3_NAME
+import random
+s3 = boto3.resource('s3')
+
+def send_answer(request):
+    if request.method == "POST":
+        uploaded_file = request.FILES['file']
+        if (uploaded_file.name[-4:] == ".txt" or uploaded_file.name[-4:] == ".pdc"
+            ) and uploaded_file.size < 1000000:
+            s3.Bucket(S3_NAME).put_object(
+            Key='{0}@{1}'.format(request.POST["task"],file_id()), Body=uploaded_file)
+
+    task_list = []
+    task_list_1 = Task.objects.all()
+    for task in task_list_1:
+        task_list.append(task.name)
+    context = {'task_list':task_list}
+    return render(request, 'pseudo_test/send_answer.html', context)
+
+
+
+
+def file_id():
+    for i in range(100):
+        can = 1
+        file_id = random.randint(10000,99999)
```

Comment:
Może jakieś hashe? Jest mniejsze prawdopodobieństwo że wylosujesz istniejące

Reply:
Wydaje mi się że jeżeli i tak sprawdzam zawsze czy nie występuje taki sam to nie ma co robić zbyt skomplikowanych. (I tak nie spodziewamy się zbyt dużej liczby ludzi chyba)

Reply:
No na pewno nie 100000

Reply:
hashe lepiej działają i wyglądają pewniej. Pomyśl jakby youtube wyglądał gdyby miał endpoint `watch?v=1242398979821`